Wrappers/Python/test/test_io.py

```diff
@@ -4,23 +4,9 @@
 import h5py
 import numpy as np
 import vtk
-from ccpi.viewer.utils.conversion import Converter
+from ccpi.viewer.utils.conversion import *
```

Comment:
Have we actually agreed to move to 'import *'? It's not something we do in CIL and we haven't previously done it in the viewer?

Reply:
I now removed * from the test files